packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -282,6 +282,7 @@ module.exports = {
     new webpack.optimize.UglifyJsPlugin({
       compress: {
         warnings: false,
+        comparisons: false,
```

Comment:
Can we add a similar comment to the one below? We need to know when to re-enable it and have links.